lib/compiler/src/beam_asm.erl

```diff
@@ -212,7 +218,7 @@ build_file(Code, Attr, Dict, NumLabels, NumFuncs, ExtraChunks, CompileInfo, Comp
 
     %% Compile all extra chunks.
 
-    CheckedChunks = [chunk(Key, Value) || {Key, Value} <- ExtraChunks],
+    CheckedChunks = [chunk(Key, Value) || {Key, Value} <- ExtraChunks1],
```

Comment:
```suggestion
    CheckedChunks = [chunk(Key, Value) || {Key, Value} <- ExtraChunks],
```

Reply:
Neat, I'll fix it together with some comments from @jhogberg